app-object.js

```diff
@@ -5,15 +5,15 @@ import {addDefaultLights} from './util.js';
 let canvas = document.getElementById('canvas');
 let context = canvas && canvas.getContext('webgl2', {
   antialias: true,
-  alpha: true,
+  alpha: false,
```

Comment:
What triggered this change?

Note that we alpha in `app` into many places, like `https://webaverse.com` and the preview system. Se need alpha on the main canvas for these effects.

Reply:
the canvas is always cleared and filled with one color, in the code I have not seen anywhere a custom cleaning of the canvas, you do not need the transparency of the canvas, it only wastes resources, are you sure you need it?

Reply:
Yes we do need the transparency. It is used in the previews.

Reply:
this is the point that the canvas is completely filled in, therefore alpha is not needed, it is better to fix a black flash than to include alpha, there will be an increase in performance